src/js/go/wasm_exec.js

```diff
@@ -106,12 +106,14 @@ const golog = require('debug')('go');
 		}
 	}
 
-	if (!global.crypto) {
+	// Tupelo note: something in our build process makes global.crypto equal to the require('crypto') below
+	// unsure what it is, but it does *not* add the getRandomValues which golang needs - so this 
+	// makes sure that function is available and expected.
```

Comment:
“and exported”?

Reply:
exported isn't exactly what I meant - I took another pass.